src/zrc/datamodel/migrations/0057_audittrail.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.0.13 on 2019-05-02 15:22
```

Comment:
Squash migrations please (delete the new ones in your branch, and create just one)